src/test/java/uk/gov/hmcts/ccd/domain/service/createcase/SubmitCaseTransactionTest.java

```diff
@@ -257,7 +258,7 @@ void shouldPersistEventWithSignificantDocument() {
             idamUser,
             caseEventDefinition,
             this.caseDetails,
-            IGNORE_WARNING);
+            IGNORE_WARNING, null);
```

Comment:
Put the `null` parameter on a new line just so it's consistent with all the other parameters.

Reply:
done